README.md

```diff
@@ -62,7 +62,9 @@ expect a response code of 200
 The SSL cert can be verified
 
 #### ``` be_served_from_origin ```
-```expect(url).to be_served_from_origin origin```
+```expect(url).to be_served_from_origin(origin, custom_headers)```
```

Comment:
Thanks, I've just reverted the merge so we can have this conversation before people start using it and everything gets harder. You raise a good point, is there a different interface you would suggest?